bin/release_chart

```diff
@@ -107,8 +107,28 @@ else #if installing from local, "unpacked" chart, then restore dependencies firs
   helm dependency build $chart
 fi
 
-msg "Installing chart"
-helmCmd="helm upgrade $release $chart --install $repoUrl $version $customFlags --force --reset-values $namespace $customVals"
-
-msg "$helmCmd"
-eval "$helmCmd"
+if [ $ACTION == 'push' ]
+then
+	msg "Packaging the chart..."
+	helm package $chart $version --destination /dev/shm/ 1>/dev/null
+	packedChartName=$(ls /dev/shm/)
+	case $CHART_REPO_URL in
+		http*)
+			msg "Pushing the chart to a custom web server..."
+			curl -v --data-binary "@/dev/shm/$packedChartName" $CHART_REPO_URL msg "Successfully pushed" || msg "Push failed"
```

Comment:
Are you missing && between pushing the chart and printing success message?

Reply:
Yes, thanks for the notion, I must have deleted accidentally. Will correct that